README.md

```diff
@@ -5,4 +5,38 @@ List of movies their actor/actress
 ![moviesList](/assets/1.PNG)
 ![movieDeatil](/assets/2.PNG)
 ![people](/assets/3.PNG)
-![peopleBio](/assets/4.PNG)
\ No newline at end of file
+![peopleBio](/assets/4.PNG)
+
+### Technologies Used :
+  * Angular-JS
+  * GitHub API
+  * Netlify.com (for hosting)
+### How to Run Locally :
+1. Clone the Repository
+     `git clone https://github.com/NJACKWinterOfCode/Moviepedia.git`
+2. cd into Ravenous
+      `cd Moviepedia`
+3.  Install angular cli      
+      ` npm install - g @angular/cli` 
+4. Create new project by command  
```

Comment:
Why creating new project?

Reply:
I did as you said how the project will run

Reply:
It also comes here @samagragupta

Reply:
Now you tell it, remove it